Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -52,22 +52,35 @@
 module Homebrew
   module_function
 
-  def audit
-    Homebrew::CLI::Parser.parse do
-      switch      "--strict"
-      switch      "--online"
-      switch      "--new-formula"
-      switch      "--fix"
-      switch      "--display-cop-names"
-      switch      "--display-filename"
+  def audit_args
+    Homebrew::CLI::Parser.new do
+      banner <<~EOS
+        Usage: brew audit [options] [<formulae>]
+
+        Check <formulae> for Homebrew coding style violations. This should be
+        run before submitting a new formula.
+
+        If no <formulae> are provided, all of them are checked.
+
+      EOS
+      switch      "--strict", description: "Run additional style checks, including Rubocop style checks."
+      switch      "--online", description: "Run additional slower style checks that require a network connection."
+      switch      "--new-formula", description: "Run various additional style checks to determine if a new formula is eligible for Homebrew. This should be used when creating new formula and implies --strict and --online."
```

Comment:
This line (and some others) is a bit too long. Maybe stick each `description:` on a new line?